tests/Repository.py

```diff
@@ -1961,6 +1954,29 @@ def testRepoVariables(self):
         for matched_repo_variable in matched_repo_variables:
             matched_repo_variable.delete()
 
+    @mock.patch("github.PublicKey.encrypt")
+    def testCreateRepoActionsSecret(self, encrypt):
+        repo = self.g.get_repo("demoorg/demo-repo-1")
+        # encrypt returns a non-deterministic value, we need to mock it so the replay data matches
+        encrypt.return_value = "M+5Fm/BqTfB90h3nC7F3BoZuu3nXs+/KtpXwxm9gG211tbRo0F5UiN0OIfYT83CKcx9oKES9Va4E96/b"
+        secret = repo.create_secret("secret_name", "secret-value", "actions")
+        self.assertIsNotNone(secret)
+
+    @mock.patch("github.PublicKey.encrypt")
+    def testCreateRepoDependabotSecret(self, encrypt):
+        repo = self.g.get_repo("demoorg/demo-repo-1")
+        # encrypt returns a non-deterministic value, we need to mock it so the replay data matches
+        encrypt.return_value = "M+5Fm/BqTfB90h3nC7F3BoZuu3nXs+/KtpXwxm9gG211tbRo0F5UiN0OIfYT83CKcx9oKES9Va4E96/b"
+        secret = repo.create_secret("secret_name", "secret-value", "dependabot")
+        self.assertIsNotNone(secret)
+
+    def testRepoGetSecretAssertion(self):
+        repo = self.g.get_repo("demoorg/demo-repo-1")
+        try:
+            repo.get_secret(secret_name="splat", secret_type="supersecret")
+        except AssertionError:
+            assert True
```

Comment:
Same here, what does this test assert?

Reply:
This test confirms that if you pass in a secret_type that isn't "dependabot" or "actions" it successfully throws an error. 

I added this because codecov flagged this as untested